config/initializers/application_controller_renderer.rb

```diff
@@ -0,0 +1,8 @@
+# Be sure to restart your server when you modify this file.
```

Comment:
Do we need this commented file? If yes, let's configure it with `ENV.fetch("HOST")` and make sure it works as expected.